src/lists/default.json

```diff
@@ -173,7 +179,6 @@
     { "name": "Aunt Holiday", "img": "characters/unsorted/Aunt_Holiday.png", "filly": false, "eqg": false, "gender": "female", "community": false },
     { "name": "Aunt Orange", "img": "characters/unsorted/Aunt_Orange.png", "filly": false, "eqg": false, "gender": "female", "community": false },
     { "name": "Aunt Lofty", "img": "characters/unsorted/Auntie_Lofty.png", "filly": false, "eqg": false, "gender": "female", "community": false },
-    { "name": "Autumn Blaze", "img": "characters/unsorted/Autumn_Blaze.png", "filly": false, "eqg": false, "gender": "female", "community": false },
```

Comment:
If you are moving Autumn_Blaze.png from unsorted over to kirins, you should also delete the one on the unsorted folder  so it doesnt use extra space